### ISSUE TITLE
我想询问如何在点击文章之后自动收起网站顶部？

### ISSUE THREAD

**Issue by lzkids** [liuyib/hexo-theme-stun#157]
## 你的反馈

<！-- 将你的反馈写在下面：-->

我配置好后，每次点击文章都需要手动收起网站顶部，很麻烦。所以问下该怎么解决？谢谢
---

**Comment by liuyib** [comment]
启用 `pjax` 并将 `pjax.avoidBanner` 设为 true

**Comment by lzkids** [comment]
> 启用 `pjax` 并将 `pjax.avoidBanner` 设为 true。
> 
> 不过启用 pjax 会带来一些小的副作用，请查看文档中的 [pjax 部分](https://theme-stun.github.io/docs/zh-CN/advanced/third-part.html#%E5%90%AF%E7%94%A8-pjax)。

问个和主题不相关的问题，好看的标签是怎么加入lssues的？

**Comment by liuyib** [comment]
> 问个和主题不相关的问题，好看的标签是怎么加入lssues的？

点击 Issue 右边的 label -> Edit labels：

![image](https://user-images.githubusercontent.com/38221479/105918969-ca80b100-606f-11eb-93df-f1d584148b1c.png)

然后点击 New label，添加你自定义的 label，然后就可以使用了。
